### ISSUE TITLE
"renew-https-cert" cannot find pinned version 0.12.1

### ISSUE THREAD

**Issue by RichardBradley** [EngineerBetter/control-tower#73]
The "renew-https-cert" daily job on my control-tower is not running since I reinstalled a few months ago.

The output just spins at:
> waiting for a suitable set of input versions
> control-tower-release - pinned version tag:0.12.1 not found

It gives the same output whether or not I pin the "control-tower-release" resource.
The version 0.12.1 is currently listed in that resource on my system, due to #72

Does anyone know how I can fix it?

Here's the job page:
![image](https://user-images.githubusercontent.com/1265780/90611282-6e7f4a00-e1fe-11ea-8035-992aaff41d53.png)

Here's the resource page:
![image](https://user-images.githubusercontent.com/1265780/90611321-7d65fc80-e1fe-11ea-9c0e-6ce739dec4e1.png)

**Comment by RichardBradley** [comment]
I think this is the same or similar issue as #56 
Last time, I ended up wiping and re-installing control-tower.

**Comment by andy-paine** [comment]
This may be an issue with Concourse. You can try forcing a lookup of that version to kick Concourse into finding it with:
`fly -t <your-target-name> check-resource -r control-tower-self-update/control-tower-release -f tag:0.12.1`

**Comment by andy-paine** [comment]
Quick update: this may be fixed by using latest `control-tower` due to this fix in Concourse 6.4.0 - https://github.com/concourse/concourse/releases#5780

We use the Github release resource to find new versions on `control-tower` and there was a bug related to pinning with partial versions in `get` steps (we do that [here](https://github.com/EngineerBetter/control-tower/blob/b2a848f2be9dba0e832fd041021bcdbe4e87875c/fly/aws_pipeline.go#L106)) which should be fixed after updating https://github.com/concourse/github-release-resource/issues/91.

Hopefully updating will be a permanent fix. If this is preventing you from updating, you could probably pull the pipeline with `fly get-pipeline -p control-tower-self-update` and fix the
```
get: control-tower-release
version:
  tag: 0.12.1
```
to be
```
get: control-tower-release
version:
  tag: 0.12.1
  id: "25952855"
  timestamp: "2020-04-28T08:47:19Z"
```
which should hopefully discover the right version

**Comment by RichardBradley** [comment]
This was fixed by the latest version via #72

Thanks very much for your help!
